packages/client/test/integration/Resends2.test.ts

```diff
@@ -428,6 +428,9 @@ describe('Resends2', () => {
     })
 
     it('decodes resent messages correctly', async () => {
```

Comment:
For these cases it would be nice to have a special `it` that skips the test properly when running in Electron.